src/metadata_validator.py

```diff
@@ -174,7 +174,7 @@ def validate_nodes(self, data_records, submission_id, scope):
 
         return validated_count
 
-    def validate_node(self, data_record):
+    def validate_node(self, data_record, submission_id, scope):
```

Comment:
submission ID should be included inside data_record, right? If so, there is no need to add submission_id parameter.